docs/javascript/v2/how--to-guides/debugging/faq.mdx

```diff
@@ -168,3 +168,44 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
 #### I get type errors related to WebRTC Stats, for example, 'Cannot find name RTCInboundRtpStreamStats'.
 
 The minimum supported version of TypeScript for the SDK is '4.4'. You could update to any version above 4.4 or if you are on an old version of TypeScript and cannot do a major upgrade you can set `skipLibCheck: true` in your tsconfig file.
+
+#### Why is my screenshare not working on mobile browsers?
+
+The screenshare API(getDisplayMedia) is not yet supported on mobile browsers. Refer [this](https://caniuse.com/?search=getDisplayMedia) for supported browsers.
+
+
+#### Can I have multiple instances of HMSRoomProvider each mapping to a different room?
+
+Starting from this [release](https://www.100ms.live/docs/javascript/v2/changelog/release-notes), you should be able to do it. Here is an example:
+
+```jsx
+const App = () => {
+    return (
+        <Fragment>
+          <HMSRoomProvider>
+            <Room1> 
+             <ConfereceApp />
+            </Room1>
+          </HMSRoomProvider>
+          <HMSRoomProvider>
+            <Room2>
+             <ConfereceApp />
+            </Room2>
+          </HMSRoomProvider>
+        </Fragment>
+    )
+}
+```
+
+Room1, Room2 can have the details of the room to which they want to point to
+
+Gotchas:
+
+- The app should have separation between the multiple HMSRoomProvider's like the example above.
+
+
+#### How can I modify noiseSuppression, echoCancellation or autoGainControl on the audio track?
+
+`await hmsActions.setAudioSettings({ advanced:[{ noiseSuppression: false}, { echoCancellation: false }, { autoGainControl: false } ]})`
```

Comment:
does this actually work? i thought these settings can be configured only at the time of getusermedia

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=796964
https://stackoverflow.com/questions/60873347/applying-constraints-to-an-audio-track-from-getusermedia

Reply:
or are we internally calling gum again and doing replacetrack?

Reply:
Yes, they do work. we shared this with circle as well. And also peeq. we get a new track when these advanced values are passed.